soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -237,6 +238,16 @@ std::unordered_map<std::string, RandomizerSettingKey> SpoilerfileSettingNameToEn
     { "World Settings:Decouple Entrances", RSK_DECOUPLED_ENTRANCES },
     { "Misc Settings:Gossip Stone Hints", RSK_GOSSIP_STONE_HINTS },
     { "Misc Settings:Hint Clarity", RSK_HINT_CLARITY },
+    { "Misc Settings:ToT Altar Hint", RSK_TOT_ALTAR_HINT },
+    { "Misc Settings:Ganondorf LA Hint", RSK_GANONDORF_LIGHT_ARROWS_HINT },
+    { "Misc Settings:Dampe's Diary Hint", RSK_DAMPES_DIARY_HINT },
+    { "Misc Settings:10 GS Hint", RSK_KAK_10_SKULLS_HINT },
+    { "Misc Settings:20 GS Hint", RSK_KAK_20_SKULLS_HINT },
+    { "Misc Settings:30 GS Hint", RSK_KAK_30_SKULLS_HINT },
+    { "Misc Settings:40 GS Hint", RSK_KAK_40_SKULLS_HINT },
+    { "Misc Settings:50 GS Hint", RSK_KAK_50_SKULLS_HINT },
+    { "Misc Settings:Warp Song Hint", RSK_WARP_SONGS_HINT },
```

Comment:
nit: I think there should be a plural here somewhere: `Warp Song Hints` or `Warp Songs Hint`. And needs to match the 3ds settings option above.